### ISSUE TITLE
Layout broken for Safari

### ISSUE THREAD

**Issue by dharasim** [DCMLab/MIDFT#26]
This is not urgent at the moment, I just wanted to record the bug.
We should remember to recommend to the users to use chrome or firefox.

<img width="1670" alt="Screenshot 2021-11-18 at 10 05 28" src="https://user-images.githubusercontent.com/19477065/142384797-fc27a39b-4190-4df0-bc7e-867a2971638d.png">

**Comment by dharasim** [comment]
<img width="1253" alt="Screenshot 2021-11-18 at 10 08 12" src="https://user-images.githubusercontent.com/19477065/142385138-1c6065e8-d7e9-4e77-8f3c-0ba6d78b2e90.png">

**Comment by heyGio** [comment]
Actually only Chrome is the one that implements all the features!
Since WebMIDI API is not implemented in Firefox ([https://bugzilla.mozilla.org/show_bug.cgi?id=836897](url) still marked as "open"), only Chrome is able to receive midi messages from a midi device.
I discussed about that with @fabianmoss a long time ago, when I've implemented this particular feature, and concluded to reccomend the user to use mainly Chrome.
